.github/CODEOWNERS

```diff
@@ -341,7 +342,7 @@
 /pkg/runtime/                           @DataDog/agent-shared-components
 /pkg/secrets/                           @DataDog/agent-shared-components
 /pkg/serializer/                        @DataDog/agent-metrics-logs
-/pkg/tagger/                            @DataDog/container-integrations
+/comp/core/tagger/                            @DataDog/container-integrations
```

Comment:
it seems that you added twice the `/comp/core/tagger/` package.
do we still have some code in `/pkg/tagger` ? if not you can remove this line.